articles/design/creating-invite-only-applications.md

```diff
@@ -1,118 +1,137 @@
-	"bodySize": -1,
-	"comment": ""
-}
+    {% endif %}
+
+            <p>If you have any issues with your account, please don’t hesitate to contact us at 1-888-AWESOMECO.</p>
+
+            <br>
+            Thanks!
+            <br>
 ```
 
-### Password Reset
-
-Once you've verified the user's email, you will need to initiate the [password change process](/connections/database/password-change). To do so, your app should make a `POST` request to Auth0's Management API.
-
-Be sure to replace the placeholder values for your [API Access Token](/api/management/v2/tokens), as well as those within the body of the call, including the callback/return URL for your app and the user's details.
-
-```har
-{
-	"method": "POST",
-	"url": "https://${account.namespace}/api/v2/tickets/password-change",
-	"httpVersion": "HTTP/1.1",
-	"cookies": [],
-	"headers": [{
-		"name": "Authorization",
-		"value": "Bearer MGMT_API_ACCESS_TOKEN"
-	}],
-	"queryString": [],
-	"postData": {
-		"mimeType": "application/json",
-		"text": "{ \"result_url\": \"YOUR_APP_CALLBACK_URL\", \"user_id\": \"USER_ID\", \"new_password\": \"secret\", \"connection_id\": \"con_0000000000000001\", \"email\": \"EMAIL\", \"ttl_sec\": 0 }"
-	},
-	"headersSize": -1,
-	"bodySize": -1,
-	"comment": ""
+```js
+function (user, context, callback) {
+  const request = require('request');
+  const userApiUrl = auth0.baseUrl + '/users/';
+
+  // This rule is only for Auth0 databases
+  if (context.connectionStrategy !== 'auth0') {
+    return callback(null, user, context);
+  }
+
+  if (user.email_verified || !user.last_password_reset) {
+    return callback(null, user, context);
+  }
+
+  // Set email verified if a user has already updated his/her password
+  request.patch({
+    url: userApiUrl + user.user_id,
+    headers: {
+      Authorization: 'Bearer ' + auth0.accessToken
+    },
+    json: { email_verified: true },
+    timeout: 5000
+  },
+  function(err, response, body) {
+    // Setting email verified isn't propagated to id_token in this
+    // authentication cycle so explicitly set it to true given no errors.
+    context.idToken.email_verified = (!err && response.statusCode === 200);
+
+    // Return with success at this point.
+    return callback(null, user, context);
+  });
 }
 ```
 
-## Summary
+Next time you send them a password reset email, it will use the "existing user" version of the template. The invitation email password reset link has a configuration TTL which defaults to 5 days, so if they don't accept the invite, it will eventually timeout and they can request another invitation. 
+
+## Customize the Password Reset UI
+
+Once the user has recieved their invitation email, they are expected to set a password for their account. You can use the [Universal Login Password Reset](/universal-login/password-reset) page. You can copy changes to address the set password experience. See [Customize Password Reset Using Query Parameters](/universal-login/password-reset) for details.
```

Comment:
Once the user clicks the link in the Invitation they will be brought to the [Universal Login Password Reset](/universal-login/password-reset) page. Here they will set a password for their account.  Since this page is used both for the forgot password workflow and for your user invites you will want to use the query parameters you defined earlier to identify the invite workflow and [customize the UI accordingly](/universal-login/password-reset)